### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by asq88** [community/community#50372] [Other Feature Feedback, Questions, & Ideas]
How can recover my Google 2FA codes?

**Comment by DNin01** [comment]
I'm assuming you lost your authenticator app. If you still have the recovery codes for your GitHub account or if you're still logged in on one of your devices, you can use it to get back in and then register a new authenticator app so you can complete two-factor authentication again.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @asq88! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
